doc/model_details/siegert_neuron_integration.ipynb

```diff
@@ -0,0 +1,272 @@
+    "\n",
+    "$$\\phi(\\mu,\\sigma)\t=\t\\frac{e^{-\\tilde{V}_{\\mathrm{th}}^{2}}}{e^{-\\tilde{V}_{\\mathrm{th}}^{2}}\\tau_{\\mathrm{ref}}+\\tau_{\\mathrm{m}}\\sqrt{\\pi}\\left(2D(\\tilde{V}_{\\mathrm{th}})+e^{-\\tilde{V}_{\\mathrm{th}}^{2}}\\int_{\\tilde{V}_{\\mathrm{th}}}^{|\\tilde{V}_{\\mathrm{r}}|}\\mathrm{erfcx}(s)ds\\right)}$$\n",
+    "\n",
+    "as a numerically safe expressions for $\\tilde{V}_{\\mathrm{r}}\\le0\\le\\tilde{V}_{\\mathrm{th}}$."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Noise-free Limit\n",
+    "\n",
+    "Even the noise-free limit $\\sigma\\ll\\mu$, where the implementation from ([Hahne et al., 2017](https://www.frontiersin.org/articles/10.3389/fninf.2017.00034/full#h10)) eventually breaks, works flawlessly. In this limit, $|\\tilde{V}_{\\mathrm{th}}|\\gg1$ as long as $\\mu\\neq V_{\\mathrm{th}}$; thus, we get both in the 'strong inhibition' and in the 'itermediate' regime $\\phi(\\mu,\\sigma)\\sim e^{-\\tilde{V}_{\\mathrm{th}}^{2}}\\approx0$ for $\\tilde{V}_{\\mathrm{th}}\\ge0$. Accordingly, the only interesting case is the 'strong excitation' regime $\\tilde{V}_{\\mathrm{th}}<0$. Since also $|\\tilde{V}_{\\mathrm{r}}|\\gg1$, the integrand $\\mathrm{erfcx}(s)$ is only evaluated at $s\\gg1$. Using the only the first term of the [asymptotic expansion](https://en.wikipedia.org/wiki/Error_function#Asymptotic_expansion)\n",
+    "\n",
+    "$$\\mathrm{erfcx}(s)=\\frac{1}{s\\sqrt{\\pi}}\\sum_{n=0}^{\\infty}(-1)^{n}\\frac{(2n-1)!!}{(2s^{2})^{n}}$$\n",
+    "\n",
+    "leads to the analytically solvable integral\n",
+    "\n",
+    "$$I(\\tilde{V}_{\\mathrm{th}},\\tilde{V}_{\\mathrm{r}})=\\int_{|\\tilde{V}_{\\mathrm{th}}|}^{|\\tilde{V}_{\\mathrm{r}}|}\\mathrm{erfcx}(s)ds\\approx\\frac{1}{\\sqrt{\\pi}}\\int_{|\\tilde{V}_{\\mathrm{th}}|}^{|\\tilde{V}_{\\mathrm{r}}|}\\frac{1}{s}ds=\\frac{1}{\\sqrt{\\pi}}\\log\\frac{|\\tilde{V}_{\\mathrm{r}}|}{|\\tilde{V}_{\\mathrm{th}}|}.$$\n",
+    "\n",
+    "Inserting this into $\\phi(\\mu,\\sigma)$ and using $\\tilde{V}_{\\mathrm{th}}\\approx\\frac{V_{\\mathrm{th}}-\\mu}{\\sigma}, \\tilde{V}_{\\mathrm{r}}\\approx\\frac{V_{\\mathrm{r}}-\\mu}{\\sigma}$ yields\n",
+    "\n",
+    "$$\\phi(\\mu,\\sigma)\\approx\\begin{cases}\n",
+    "0 & \\mu\\le V_{\\mathrm{th}}\\\\\n",
+    "\\frac{1}{\\tau_{\\mathrm{ref}}+\\tau_{\\mathrm{m}}\\log\\frac{\\mu-V_{\\mathrm{r}}}{\\mu-V_{\\mathrm{th}}}} & \\mu>V_{\\mathrm{th}}\\end{cases}$$\n",
+    "\n",
+    "[as it should](https://neuronaldynamics.epfl.ch/online/Ch8.S3.html). Thus, as long as the numerical solution of the integral $\\frac{1}{\\sqrt{\\pi}}\\int_{|\\tilde{V}_{\\mathrm{th}}|}^{|\\tilde{V}_{\\mathrm{r}}|}\\frac{1}{s}ds$ is precise, the deterministic limit is also numerically safe.\n",
+    "\n",
+    "Looking at the difference between $\\mathrm{erfcx}(s)$ and the first order asymptotics, we see that the noise free limit is already relevant for $|\\tilde{V}_{\\mathrm{th}}|\\approx O(100)$:"
```

Comment:
```suggestion
    "Looking at the difference between $\\mathrm{erfcx}(s)$ and the first order asymptotics, we see that the noise free limit is already relevant for $\\left|\\tilde{V}_{\\mathrm{th}}\\right|\\approx O(100)$:"
```

Reply:
"we see that the noise free limit is already relevant for O(100)" sorry for not understanding this, doesn't the plot show that the approximation is good for (say) argument > 100, and it breaks down for smaller values? Perhaps I don't understand how this conclusion about the noise-free limit follows from the goodness of the approximation.